Symfony/CS/Fixer/Symfony/PhpdocIndentFixer.php

```diff
@@ -55,6 +55,24 @@ public function fix(\SplFileInfo $file, $content)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'Docblocks should have the same indentation as the documented subject.';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getPriority()
+    {
+        // should be run before all other phpdoc fixers
```

Comment:
It's before those fixers because their ordering depends on other fixers that depend on this. :)